docs/docs/Release_Notes/3.0.0/3.0.0.md

```diff
@@ -3,11 +3,19 @@
 To install or upgrade the SDK, refer to instructions [here](/Getting_Started.md#installation).
 
 ## New & Improved
-* Added a `scratch_path` property on the [RemoteHost](/References/Classes/#remotehost) object for storage and debugging purposes.
+* Added the ability to define snapshot parameters in a [Snapshot Parameters Definition](/References/Schemas_and_Autogenerated_Classes.md#snapshotparametersdefinition-schema).
+	* Provide end-users with configurable options prior to taking a snapshot.
+	* The options selected are provided as input to pre/post-snapshot functions.
+
+* Added a `scratch_path` property on the [RemoteHost](/References/Classes/#remotehost) object which can be used as:
+	* A location to store small amounts of persistent data.
+	* A location to mount VDB data.
 More details about `scratch_path` can be found [here](/Best_Practices/Scratch_Paths.md)
 
 ## Breaking Changes
 
 * Added a new required schema [Snapshot Parameters Definition](/References/Schemas_and_Autogenerated_Classes.md#snapshotparametersdefinition-schema).
```

Comment:
can we add a `For more information and detailed steps to detect and make changes.` link for each of the breaking changes since they point to different sections in the `/Release_Notes/3.0.0/3.0.0_Breaking_Changes` page